skins/laika/templates/partials/back_link.html.twig

```diff
@@ -0,0 +1,5 @@
+<div class="back-block col-xs-12 col-sm-12 col-md-12">
+	<ul class="list-menu link-back list-unstyled">
+		<li><a href="{$ path( '/' ) $}" class="">{$ 'back_to_donation' | trans $}</a></li>
+	</ul>
+</div>
```

Comment:
I think this template might have slipped into this PR by accident. The laika version of `templates/page_layouts/base.html.twig` does not use it.